run.sh

```diff
@@ -42,7 +42,7 @@ function mirror(){
     # delete all hidden github pull request refs
     git for-each-ref \
       --format='delete %(refname)' \
-      "${IGNORE_REFS_PATTERN}" \
+      ${IGNORE_REFS_PATTERN} \
```

Comment:
If a ref ends on *, this will expand to all fils in the local directory.

```suggestion
      "${IGNORE_REFS_PATTERN[@]}" \
```

Reply:
Yes, your solution is better.